sci-biology/open-ephys-gui/open-ephys-gui-9999.ebuild

```diff
@@ -65,8 +68,8 @@ src_configure() {
 }
 
 src_install() {
-	dodir opt/open-ephys-"${Suffix}"/ lib/udev/rules.d/
-	cp -R "${BUILD_DIR}"/Gentoo/* "${ED}"/opt/open-ephys-"${Suffix}"/
+	dodir opt/open-ephys-"${Suffix}"/ lib/udev/rules.d/ || die
```

Comment:
`dodir` is a built-in function, so it doesn't need the `|| die`. We only need that for external commands. Not that it hurts to have it here.

Reply:
Ah, got it! fixed.